serv/algo.py

```diff
@@ -57,14 +57,22 @@ def save_data(self, embed_file, meta_file):
               - status: Boolean
         '''
         status = True
-
+        
         if not (embed_file and meta_file):
             return False
-
-        if embed_file: pkl.dump(self.embed_data, open(embed_file, "wb"))
+        # breakpoint()
```

Comment:
Can you split the commits to be atomic features. So have separate commit for date -tile and title, another for key error bug fix and one for SIGEXIT corruption.